src/middleware/secure-headers/secure-headers.ts

```diff
@@ -162,6 +165,7 @@ export const NONCE: ContentSecurityPolicyOptionHandler = (ctx) => {
 export const secureHeaders = (customOptions?: SecureHeadersOptions): MiddlewareHandler => {
   const options = { ...DEFAULT_OPTIONS, ...customOptions }
   const headersToSet = getFilteredHeaders(options)
+
```

Comment:
Sorry, one thing. Can you remove the change for this one line?

Reply:
I am sorry for the inconvenience.
Everything is ready